.github/ISSUE_TEMPLATE/edit_this_page.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Edit this page
+about: Many thanks for your feedback but the page you are trying to edit is not available publicly. Please fill out the information below and Datadog Documentation team will take care of updating the corresponding documentation page.
+
+---
+
+**Documentation page to update**:
```

Comment:
are we asking for them to put the URL? let's ask explicitly for a URL

Reply:
```suggestion
**Documentation page URL to update**:
```